util/cache2.cc

```diff
@@ -421,6 +423,33 @@ class ShardedLRUCache2 : public Cache {
 
   } // ShardedLRUCache2::PurgeExpiredFiles
 
+  // Walk all cache entries, calling functor Acc
+  //  Writes list of open files to log record
```

Comment:
The "Writes list of open files to log record" looks to be a holdover from the previous COW-specific implementation